rdrf/rdrf/export/model_exporters.py

```diff
@@ -0,0 +1,72 @@
+import os
+from rdrf import models
+from registry.patients import models as patientmodels
+from .catalogue import ModelExporterCatalogue
+from .exporters import ModelExporter
+
+
+# Catalogue with generic exporters
+catalogue = ModelExporterCatalogue()
+
+# Registry specific exporters. Querysets filtered by registry code.
+registry_catalogue = ModelExporterCatalogue()
+
+class RegistryExporter(ModelExporter):
+    @property
+    def queryset(self):
+        return self.model.objects.filter(code=self.exporter_context['registry_code'])
+
+
+class RegistryFormExporter(ModelExporter):
+    @property
+    def queryset(self):
+        return self.model.objects.filter(registry__code=self.exporter_context['registry_code'])
+
+
+class AdjudicationDefinitionExporter(ModelExporter):
+    @property
+    def queryset(self):
+        return self.model.objects.filter(registry__code=self.exporter_context['registry_code'])
+
+
+class SectionExporter(ModelExporter):
+    @staticmethod
+    def get_sections(registry_code):
+        registry = models.Registry.objects.get(code=registry_code)
+        for form in models.RegistryForm.objects.filter(registry=registry):
+            for section_code in form.get_sections():
+                yield models.Section.objects.get(code=section_code)
```

Comment:
I think I did add a property called section_models on RegistryForm to do this  

for section_model in form.section_models:
   yield section_model